util/camelcase_test.go

```diff
@@ -0,0 +1,11 @@
+package util
+
+import "testing"
+import "strings"
+
```

Comment:
Add a more test cases for `ToUnderScore ` here.
For example: _requestID, _request, Request_ID, ... etc

Reply:
OK